adafruit_circuitplayground/express.py

```diff
@@ -563,24 +563,25 @@ def red_led(self, value):
         self._led.value = value
 
     @staticmethod
-    def _sine_sample(length):
-        tone_volume = (2 ** 15) - 1
+    def _sine_sample(length, volume=None):
+        if volume is None:
+            volume = 0
+        tone_volume = (2 ** 15) + volume
         shift = 2 ** 15
         for i in range(length):
             yield int(tone_volume * math.sin(2*math.pi*(i / length)) + shift)
 
-    def _generate_sample(self, length=100):
+    def _generate_sample(self, length=100, volume=None):
```

Comment:
As written, won't this cause the *first* volume used to apply to all samples until a restart?